### ISSUE TITLE
Android equivalent to Sign In with Apple

### ISSUE THREAD

**Issue by antonioberetini** [CodetrixStudio/CapacitorGoogleAuth#164]
Is this the equivalent to Sign in with Apple? On apple, users can signin in by a single tap where they define email and by clicking continue it uses biometrics if available to authenticate, with that you are all set. I am looking for Android alternative to that. Is this the same approach?

**Comment by beestatechristos** [comment]
No. It only let users to sign in using their Google Account. No biometrics.
